### PR TITLE
add PUT Request and add EditMember Form

### PR DESCRIPTION
**Pull request by Fabi911**

### CODE REVIEW

File: frontend/src/Pages/MemberDetail.tsx [hunk 1]
```diff
@@ -27,13 +33,45 @@ export default function MemberDetail() {
...
+        <DetailsBox>
+            <BackLink to={"/members"}><ArrowBackIcon fontSize="large"/></BackLink>
+            <MemberBox>
+                <h1>{name} {lastName}</h1>
+                <span>Mitgliedsnr.: {memberId}</span>
+                <span>Adresse: <br/>{address.street}<br/> {address.zip} {address.city}</span>
+                <span>Tel: {phoneNumber}</span>
+                <span>E-Mail: {email}</span>
+                <span>Geburtstag: {new Date(birthday).toLocaleDateString()}</span>
+<EditButton onClick={() => setModal(true)}><EditNoteIcon fontSize="large"/> </EditButton>
```
Comment (rinaehyun) [inline frontend/src/Pages/MemberDetail.tsx]: Align this line please

**Comment by rinaehyun** [comment]
Generally, looks good to me.
Can I ask you one thing? If the user wants to update an Item, which does not exist in DB, then what happens? There would be an exception handling or..?

**Comment by Fabi911** [comment]
> Generally, looks good to me. Can I ask you one thing? If the user wants to update an Item, which does not exist in DB, then what happens? There would be an exception handling or..?

at the moment this function is only possible in the detail view of a member, the id is already checked.
But if it happens that the id is not available, the member is saved as a new member. See @MemberService.java line 36.

**Comment by rinaehyun** [comment]
Okay :) then everything looks good to me :)

**Review by rinaehyun — APPROVED** [review]
good :)

**Review by rinaehyun — APPROVED** [review]
Approved :)

**Pull request merged**